contrib/htop/template.py

```diff
@@ -1,14 +1,13 @@
 pkgname = "htop"
-pkgver = "3.2.2"
+pkgver = "3.3.0"
 pkgrel = 0
 build_style = "gnu_configure"
-configure_args = ["--enable-unicode", "--enable-sensors"]
```

Comment:
why remove that?

Reply:
they're both the default afaict (see log)

though i prefer leaving those in anyway

Reply:
it's still good to be explicit about default flags because it means things will actually properly fail instead of silently disabling the feature when something is wrong

Reply:
yep, that's why i like them :)

Reply:
I copied that from void but just realized it's unneeded since it auto-detects both during configure:
```
  (Linux) sensors:           yes
...
  unicode:                   yes
```
sooo should I leave the flags in anyway or nah?